app/views/users/mfa_selection/index.html.erb

```diff
@@ -4,6 +4,25 @@
 
 <p class="maxw-mobile-lg margin-bottom-0"><%= @presenter.intro %></p>
 
+<h2 class="margin-top-2 margin-bottom-1">
+  <%= t('headings.account.two_factor') %>
+</h2>
+
+<div class="grid-row grid-gap">
+  <div class="tablet:grid-col">
+    <ul class="usa-icon-list">
+      <% @presenter.options.each do |option| %>
+        <% if option.mfa_configuration_count > 0 %>
+          <%= render(option) do %>
```

Comment:
What does the `render(option)` do here?

Reply:
Nothing, apparently!

To be honest I borrowed and modified that list from this block in the `simple_form_for` block

`<div class="margin-bottom-4">
    <fieldset class="margin-0 padding-0 border-0">
      <legend class="margin-bottom-2 usa-sr-only"><%= @presenter.intro %></legend>
      <%= hidden_field_tag 'two_factor_options_form[selection][]', nil %>
      <% @presenter.options.each do |option| %>
        <%= render(option) do %>
          <%= render partial: 'partials/multi_factor_authentication/mfa_selection',
                     locals: { option: option } %>
        <% end %>
      <% end %>
    </fieldset>
  </div>
`

I took that `<%= render(option) do %>` out of here and the only change was the pink is now gone from FT Unlock on Mac Chrome.

Removing it from the block in question fixes the `<li> vs <lg-webauthn-input>` problem in your comment above.

Reply:
Yeah, the `render` in the options list is a way for individual presenters to control how the input field is shown (e.g. [WebAuthn input](https://github.com/18F/identity-idp/blob/096da80c91aa3a8058bc34996390335fb7135f6e/app/presenters/two_factor_authentication/webauthn_selection_presenter.rb#L7-L9)), so explains the weirdness with `lg-webauthn-input`, and we shouldn't need it here.